raw-data/FEATURES/special-feature-table.csv

```diff
@@ -62,3 +62,7 @@ tʃɾ,0074+0283+027E,0,-,-,-,-,+,-,-,+,-,+,-,-,-,-,0,0,+,-,+,+,-,0,0,0,0,0,-,-,-
 dʒɾ,0064+0292+027E,0,-,-,-,-,+,-,-,+,-,+,-,-,-,-,0,0,+,-,+,+,-,0,0,0,0,0,-,-,+,-,-,-,-,-,-,-,-
 ntʃɾ,006E+0074+0283+027E,0,-,-,-,-,+,"+,-",-,+,-,+,-,"+,-",-,-,0,0,+,-,+,+,-,0,0,0,0,0,-,-,"+,-",-,-,-,-,-,-,-,-
 ŋmkpɾ,014B+006D+006B+0070+027E,0,-,-,-,-,+,"+,-",-,-,-,"-,+",-,"+,-",-,+,-,-,+,+,-,-,+,+,-,-,-,0,-,-,"+,-",-,-,-,-,-,-,-,-
+kǀʰʼ,006B+01C0+02B0+02BC,0,-,-,-,-,+,-,-,-,-,-,-,-,-,-,0,0,"-,+",+,+,-,"+,-",+,-,-,-,0,-,-,-,-,"-,+","+,-",-,-,"-,+",-,+
+kǁʰʼ,006B+01C1+02B0+02BC,0,-,-,-,-,+,-,-,-,-,-,-,-,"-,+",-,0,0,"-,+",+,-,-,"+,-",+,-,-,-,0,-,-,-,-,"-,+","+,-",-,-,"-,+",-,+
+kǃʰʼ,006B+01C3+02B0+02BC,0,-,-,-,-,+,-,-,-,-,-,-,-,-,-,0,0,"-,+",+,-,-,"+,-",+,-,-,-,0,-,-,-,-,"-,+","+,-",-,-,"-,+",-,+
+k‼ʰʼ,006B+203C+02B0+02BC,0,-,-,-,-,+,-,-,-,-,-,-,-,-,-,0,0,"-,+",-,-,-,"+,-",+,-,-,-,0,-,-,-,-,"-,+","+,-",-,-,"-,+",-,+
```

Comment:
it's hard to get aspirated ejective clicks correct automatically so these get special-cased.